.changes/snapshot-migration-v3age-zeroize.md

```diff
@@ -0,0 +1,13 @@
+---
+
+"iota-stronghold": minor
+"stronghold-engine" : minor
+"stronghold-runtime" : minor
+"vault" : minor
+"snapshot" : minor
```

Comment:
The last 2 are not managed by covector, see the list here https://github.com/iotaledger/stronghold.rs/blob/dev/.changes/config.json#L73